examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -434,6 +435,38 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             // unrecognized map type
             break;
         }
+      },
+
+      pinMapOneTweetQuery: function(pinid){
+
+          var setpinfilter = [{
+              field: "id",
+              relation: "in",
+              values: pinid
+          }];
+
+          //sql
+          var pinDBquery = (JSON.stringify({
+              dataset:"twitter.ds_tweet",
+              filter: setpinfilter,
+              select:{
+                  order: ["-create_at"],
+                  limit: defaultSamplingSize,
```

Comment:
Actually, I am not sure why order and limit are required. This query is supposed to return only one result. Right?

Reply:
Yep, it could be deleted I think.

Reply:
I test and find these fields are all required otherwise the cloudberry will give errors. So although we may not need to specify it, we have to do write that.

Reply:
So these fields can not be deleted.